src/commands/apps-list.ts

```diff
@@ -0,0 +1,60 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+// TODO(caot): Replace with proper import
+import Table = require("cli-table");
+
+import * as Command from "../command";
+import * as getProjectId from "../getProjectId";
+import { listFirebaseApps } from "../management/list";
+import { AppMetadata, AppPlatform } from "../management/metadata";
+import * as FirebaseError from "../error";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+
+const NOT_SPECIFIED = clc.yellow("[Not specified]");
+
+function logAppsList(apps: AppMetadata[]): void {
+  if (apps.length > 0) {
+    const tableHead = ["App Display Name", "App ID", "Platform"];
+    const table = new Table({ head: tableHead, style: { head: ["green"] } });
+    apps.forEach(({ appId, displayName, platform }) => {
+      table.push([displayName || NOT_SPECIFIED, appId, platform]);
+    });
+
+    logger.info(table.toString());
+  } else {
+    logger.info(clc.bold("No apps found."));
+  }
+}
+
+module.exports = new Command("apps:list [platform]")
+  .description(
+    "list the registered apps of a Firebase project\n\n" +
+      "Arguments:\n" +
+      "[platform] IOS, ANDROID or WEB (case insensitive)"
+  )
+  .before(requireAuth)
+  .action(
+    async (platform: AppPlatform | string = "", options: any): Promise<AppMetadata[]> => {
```

Comment:
am I right in assuming that this function will in practice only every actually receive a `string` (which may be empty)? If so, why not:

1. have this function signature be `platform: string = ""`
2. have a helper function that takes a `string` and returns an `AppPlatform`

Reply:
Thanks :+1:

Reply:
Added `getAppPlatform()` to transform `string` to `AppPlatform`